Hardcore.lua

```diff
@@ -721,6 +756,33 @@ function Hardcore:CHAT_MSG_ADDON(prefix, datastr, scope, sender)
 	if COMM_NAME == prefix then
 		-- Get the command
 		local command, data = string.split(COMM_COMMAND_DELIM, datastr)
+		if DEPRECATED_COMMANDS[command] then return end
+
+		-- Handle weird commands
+		if alert_msg_time[command] == nil then
+			local debug_info = {command, data, sender}
+			table.insert(Hardcore_Settings.debug_log, debug_info)
+			alert_msg_time.UNKNOWN[sender] = time()
+			-- Display that someone is trying to send unknown messages; notifies mods to look at saved_vars and remove player from guild
+			if monitor_msg_throttle.UNKNOWN[sender] == nil or (time() - monitor_msg_throttle.UNKNOWN[sender] > THROTTLE_DURATION) then
+				Hardcore:Monitor("|cffFF0000Received unknown messages from " .. sender .. ".")
+				monitor_msg_throttle.UNKNOWN[sender] = time()
+			end
+			return
+		end
```

Comment:
What's the benefit of this? If someone is spamming silent MSGs and nothing is handling it, I doubt it would cause lag. Maybe we should only handle known commands

Reply:
I believe Blizzard's chat service will throttle the user spamming before it reaches our clients

Reply:
I agree; after looking into it, I don't think there is any risk here.  Originally I just wanted a way to log weird messages but I think its fine to remove.